plugins/modules/ovirt_disk.py

```diff
@@ -347,6 +347,25 @@ def _search_by_lun(disks_service, lun_id):
     return res[0] if res else None
 
 
+def create_transfer_connection(module, transfer, context, connect_timeout=10, read_timeout=60):
```

Comment:
It can be useful to expose the timeouts to the user so they can override the timeouts if needed.

Reply:
yeah thought about that but we have default timeout 3min and not sure if it would be best to wait that long

Reply:
I would this to future work.